applications/serial_lte_modem/doc/SOCKET_AT_commands.rst

```diff
@@ -540,30 +540,31 @@ Syntax
 
     * ``<value>`` is an integer that indicates the packet data network ID to bind to.
 
-  * ``50`` - :c:macro:`SO_RAI_NO_DATA` (set-only).
```

Comment:
Just a comment: I think that it makes sense to remove these from the documentation. Given that these are deprecated, we will map the old values to new values in SLM after the 2.6.0 release.